src/intro/intro_readme.md

```diff
@@ -1,4 +1,4 @@
-![header](../intro/img/intro_header.png)
+![header](./img/intro_header.png)
```

Comment:
we have to revert to the previous code:

```
![header](../intro/img/intro_header.png)
```
There is two ways to see the image/access this page:
* (1) www.manual.grid.tf
* (2) https://manual.grid.tf/intro/intro_readme.html

With ./img..., you can't access the image from the first link (1).
That is why we put ../intro/img/...
This way, the picture is there when you enter the website with (1) and when you click on the link to access (2)